### ISSUE TITLE
Whitepaper

### ISSUE THREAD

**Issue by Harryman** [Harryman/hashd#13]
The original [draft](https://hashd.in/hashd-in-draft0/) is too wide ranging to make a tight paper describing the underlying datastructures and incentives. 

This is not a priority and would be best done after we have some real world data on the network's operation. We need to set our project apart from most by having real data and product(PoC) before paper.

**Comment by das-sein** [comment]
I would like to advocate for some kind of yellow or white paper on some of the details of the protocol to be fleshed out. If we can formalize certain procedures within the protocol, we can then apply BAN logic later on to create assertions of trust and privacy.

**Comment by Harryman** [comment]
Yeah working on a revised whitepaper for the core spec, I think yellow paper after working PoC implementation
